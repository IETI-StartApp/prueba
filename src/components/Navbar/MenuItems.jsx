import React from 'react';
import List from '@material-ui/core/List';
import { ListItem, ListItemIcon, ListItemText } from '@material-ui/core/';
import LabelImportantIcon from '@material-ui/icons/LabelImportant';
import { useStyles } from './styles';

export default function MenuItems() {

    const classes = useStyles();
    const items = [
        {text:"Opción 1", icon: <LabelImportantIcon />},
        {text:"Opción 2", icon: <LabelImportantIcon />},
        {text:"Opción 3", icon: <LabelImportantIcon />},
        {text:"Opción 4", icon: <LabelImportantIcon />},
        {text:"Opción 5", icon: <LabelImportantIcon />}
    ];

    return (
        <List component='nav'>
            {
                items.map( (item) => (
                    <ListItem button className={classes.mainColor} key={item.text}>
                        <ListItemIcon>
                            {item.icon}
                        </ListItemIcon>
                        <ListItemText primary={item.text} />
                    </ListItem>
                ))
            }
        </List>
    );

}