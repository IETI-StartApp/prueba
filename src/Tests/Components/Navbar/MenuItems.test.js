import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import React from 'react';
import MenuItems from '../../../components/Navbar/MenuItems';

describe('Testing menu items', () => {

    const wrapper = shallow(<MenuItems />)

    test('should load menu items', () => {
        expect(wrapper).toMatchSnapshot();
    })
    
})
