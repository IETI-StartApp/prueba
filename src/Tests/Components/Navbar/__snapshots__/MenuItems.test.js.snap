// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing menu items should load menu items 1`] = `
<WithStyles(ForwardRef(List))
  component="nav"
>
  <WithStyles(ForwardRef(ListItem))
    button={true}
    className="makeStyles-mainColor-3"
    key="Opción 1"
  >
    <WithStyles(ForwardRef(ListItemIcon))>
      <Memo([object Object]) />
    </WithStyles(ForwardRef(ListItemIcon))>
    <WithStyles(ForwardRef(ListItemText))
      primary="Opción 1"
    />
  </WithStyles(ForwardRef(ListItem))>
  <WithStyles(ForwardRef(ListItem))
    button={true}
    className="makeStyles-mainColor-3"
    key="Opción 2"
  >
    <WithStyles(ForwardRef(ListItemIcon))>
      <Memo([object Object]) />
    </WithStyles(ForwardRef(ListItemIcon))>
    <WithStyles(ForwardRef(ListItemText))
      primary="Opción 2"
    />
  </WithStyles(ForwardRef(ListItem))>
  <WithStyles(ForwardRef(ListItem))
    button={true}
    className="makeStyles-mainColor-3"
    key="Opción 3"
  >
    <WithStyles(ForwardRef(ListItemIcon))>
      <Memo([object Object]) />
    </WithStyles(ForwardRef(ListItemIcon))>
    <WithStyles(ForwardRef(ListItemText))
      primary="Opción 3"
    />
  </WithStyles(ForwardRef(ListItem))>
  <WithStyles(ForwardRef(ListItem))
    button={true}
    className="makeStyles-mainColor-3"
    key="Opción 4"
  >
    <WithStyles(ForwardRef(ListItemIcon))>
      <Memo([object Object]) />
    </WithStyles(ForwardRef(ListItemIcon))>
    <WithStyles(ForwardRef(ListItemText))
      primary="Opción 4"
    />
  </WithStyles(ForwardRef(ListItem))>
  <WithStyles(ForwardRef(ListItem))
    button={true}
    className="makeStyles-mainColor-3"
    key="Opción 5"
  >
    <WithStyles(ForwardRef(ListItemIcon))>
      <Memo([object Object]) />
    </WithStyles(ForwardRef(ListItemIcon))>
    <WithStyles(ForwardRef(ListItemText))
      primary="Opción 5"
    />
  </WithStyles(ForwardRef(ListItem))>
</WithStyles(ForwardRef(List))>
`;
